pallets/manta-pay/src/lib.rs

```diff
@@ -557,10 +557,15 @@ pub mod pallet {
             let (more_receivers, receivers) =
                 Self::pull_receivers(*checkpoint.receiver_index, max_receivers);
             let (more_senders, senders) = Self::pull_senders(checkpoint.sender_index, max_senders);
+            let senders_receivers_total = (0..=255)
```

Comment:
I still don't quite get why this is better, than just making a separate storage item to keep track of the utxo set size.